integrations/docker/images/chip-cert-bins/Dockerfile

```diff
@@ -217,63 +217,63 @@ SHELL ["/bin/bash", "-c"]
-            && mv out/linux-x64-all-clusters-minimal-ipv6only/chip-all-clusters-minimal-app out/chip-all-clusters-minimal-app \
-            && mv out/linux-x64-bridge-ipv6only/chip-bridge-app out/chip-bridge-app \
-            && mv out/linux-x64-tv-app-ipv6only/chip-tv-app out/chip-tv-app \
-            && mv out/linux-x64-tv-casting-app-ipv6only/chip-tv-casting-app out/chip-tv-casting-app \
-            && mv out/linux-x64-light-ipv6only/chip-lighting-app out/chip-lighting-app \
-            && mv out/linux-x64-thermostat-ipv6only/thermostat-app out/thermostat-app \
-            && mv out/linux-x64-ota-provider-ipv6only/chip-ota-provider-app out/chip-ota-provider-app \
-            && mv out/linux-x64-ota-requestor-ipv6only/chip-ota-requestor-app out/chip-ota-requestor-app \
-            && mv out/linux-x64-lock-ipv6only/chip-lock-app out/chip-lock-app \
-            && mv out/linux-x64-simulated-app1-ipv6only/chip-app1 out/chip-app1 \
-                ;; \
-        "linux/arm64")\
-            set -x \
-            && source scripts/activate.sh \
-            && scripts/build/build_examples.py \
-            --target linux-arm64-all-clusters-ipv6only \
-            --target linux-arm64-all-clusters-minimal-ipv6only \
-            --target linux-arm64-bridge-ipv6only \
-            --target linux-arm64-tv-app-ipv6only \
-            --target linux-arm64-tv-casting-app-ipv6only \
-            --target linux-arm64-light-ipv6only \
-            --target linux-arm64-thermostat-ipv6only \
-            --target linux-arm64-ota-provider-ipv6only \
-            --target linux-arm64-ota-requestor-ipv6only \
-            --target linux-arm64-lock-ipv6only \
-            --target linux-arm64-simulated-app1-ipv6only \
-            build \
-            && mv out/linux-arm64-all-clusters-ipv6only/chip-all-clusters-app out/chip-all-clusters-app \
-            && mv out/linux-arm64-all-clusters-minimal-ipv6only/chip-all-clusters-minimal-app out/chip-all-clusters-minimal-app \
-            && mv out/linux-arm64-bridge-ipv6only/chip-bridge-app out/chip-bridge-app \
-            && mv out/linux-arm64-tv-app-ipv6only/chip-tv-app out/chip-tv-app \
-            && mv out/linux-arm64-tv-casting-app-ipv6only/chip-tv-casting-app out/chip-tv-casting-app \
-            && mv out/linux-arm64-light-ipv6only/chip-lighting-app out/chip-lighting-app \
-            && mv out/linux-arm64-thermostat-ipv6only/thermostat-app out/thermostat-app \
-            && mv out/linux-arm64-ota-provider-ipv6only/chip-ota-provider-app out/chip-ota-provider-app \
-            && mv out/linux-arm64-ota-requestor-ipv6only/chip-ota-requestor-app out/chip-ota-requestor-app \
-            && mv out/linux-arm64-lock-ipv6only/chip-lock-app out/chip-lock-app \
-            && mv out/linux-arm64-simulated-app1-ipv6only/chip-app1 out/chip-app1 \
-            ;; \
-        *) ;; \
+    "linux/amd64") \
+    set -x \
```

Comment:
All these whitespace changes seem to make things a bit less readable...  Why are they here?

Reply:
Ah yes, thanks for raising this, I think my editor that's been configured for some other project.

Reply:
Pushed a new commit